acra/configuring-maintaining/general-configuration/acra_keymaker.md

```diff
@@ -0,0 +1,178 @@
+---
+title: AcraKeymaker
+bookCollapseSection: true
+---
+
+# AcraKeymaker
+
+## Command line flags
+
+### General flags
+
+* `--client_id=<id>`
+  
+  Use provided client ID as identifier to generate keys or keypairs.
+
+* `--tls_cert=<path>`
+  
+  Path to TLS certificate to use as client_id identifier. Works only when passed empty ClientID: `--client_id=""`.
+
+* `--tls_identifier_extractor_type={distinguished_name|serial_number}`
+  
+  Decide which field of TLS certificate to use as ClientID.
+
+  * `distinguished_name` — (default) certificate Distinguished Name (DN).
+  * `serial_number` — certificate serial number.
```

Comment:
“(default) certificate Distinguished Name” reads like something you would choose during certificate issuance. It would be nice to tack “(default)” at the end but you already put “(DN)” there, so I guess you can just note the default later.

```suggestion
  * `distinguished_name` — certificate Distinguished Name (DN)
  * `serial_number` — certificate serial number
  
By default certificate Distinguished Name is used as client ID.
```